airflow/utils/db.py

```diff
@@ -986,3 +988,47 @@ def check(session=None):
     """
     session.execute('select 1 as is_alive;')
     log.info("Connection successful.")
+
+
+@enum.unique
+class DBLocks(enum.IntEnum):
+    """
+    Cross-db Identifiers for advisory global database locks.
+
+    Postgres uses int64 lock ids so we use the integer value, MySQL uses names, so we use the ``_name_``
+    field.
+    """
+
+    INIT = enum.auto()
+    MIGRATIONS = enum.auto()
```

Comment:
Any reason we need both `init` and `migrations` as `initdb()` runs `upgradedb()` too?

Reply:
Yeah, I wasn't sure. Previously there were _three_ separate locks: one for upgrade, one for reset, and one for the "DAG sync" part of init.

This init lock could perhaps be dropped entirely, yes

Reply:
Fixed in f18e2cdf2